test/common/http/codec_impl_fuzz.proto

```diff
@@ -87,13 +87,21 @@ message Action {
   }
 }
 
+message Http1ClientSettings {
+  bool allow_absolute_url = 1;
+  bool accept_http_10 = 2;
+  string default_host_for_http_10 = 3;
+}
```

Comment:
since we're going down this route, this should have a bool for enable trailers as well so that they're independent?

Reply:
I think we can follow the same pattern as for H2 (single type for H1 settings), no?

Reply:
Yea, though that could cause churn in the existing corpus. I was pointing out the fact that trailers encoding is effectively disabled for the client given this code but not for the server.